demos/device-shadow-for-aws-iot-embedded-sdk/shadow_demo_helpers.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * AWS IoT Device SDK for Embedded C V202009.00
```

Comment:
Can you please change this to `FreeRTOS V202010.00` in all of the new header and source files.

Reply:
ok~